front/js/order.es6

```diff
@@ -16,17 +19,31 @@
     $(DOM.$order).on('keyup', 'textarea', storeInput);
     $(DOM.$order).on('input', DOM.productCount, helpers.debounce(changeProductCount, 300));
     $(DOM.$order).on('click', DOM.remove, removeProduct);
+    $(DOM.submitBtn).on('click', DOM.submit, submitOrder);
   }
 
-  const getProductId = $target => $target.closest('.js-product-row').data('product-id');
+  const getProductId = $target => $target.closest(DOM.productRow).data('product-id');
+  const getProductCount = $target => $target.find(DOM.productCount).val();
+  const getProductName = $target => $.trim($target.find('a').text());
```

Comment:
class such as `HtmlProduct` might be perfect here. But maybe it's overkill for js